opendatahub-io/odh-dashboard
# Custom serving runtimes: fix drag problems, add deleting and editing features

Closes #1083 

## Description


1. Fix some drag-and-drop issues and remove the debounce for it.
2. Add the "delete" dropdown in the kebab toggle. When you click on it, you can open a deleting modal to delete the serving runtimes
<img width="1791" alt="Screenshot 2023-04-26 at 9 20 23 PM" src="https://user-images.githubusercontent.com/37624318/234588046-9a9aea36-cbb1-4701-ab25-b238281970e6.png">
3. Add the "edit" dropdown in the kebab toggle. When you click on it, it will take you to the same page as you adding the serving runtimes, but it will read the current information and map them in the code editor so you can update it.
<img width="1791" alt="Screenshot 2023-04-26 at 9 23 00 PM" src="https://user-images.githubusercontent.com/37624318/234588850-39f2f2f5-2c34-4758-b4fe-af37d438519c.png">

## How Has This Been Tested?



1. Try to drag and drop to see if there are any bugs
2. Try to create a serving runtime and delete it
3. Try to create a serving runtime and edit it, you can add the `openshift.io/display-name` field in the annotation so it will show the display name. Otherwise, it will show the metadata name.

## Test Impact


Will add tests after everything is done.

## Request review criteria:
